pkg/kdump/kdump-view.jsx

```diff
@@ -480,7 +480,7 @@ export class KdumpPage extends React.Component {
         }
         return (
             <Page>
-                <PageSection variant={PageSectionVariants.light}>
+                <PageSection variant={PageSectionVariants.light} className="pf-m-padding">
```

Comment:
That works but it's kinda hacky. The correct way it to make it clear, that's a header for the page. You can do that like

```
g[kkoukiou@sequioa cockpit]$ git diff
diff --git pkg/kdump/kdump-view.jsx pkg/kdump/kdump-view.jsx
index b65548919..621238835 100644
--- pkg/kdump/kdump-view.jsx
+++ pkg/kdump/kdump-view.jsx
@@ -479,15 +479,18 @@ export class KdumpPage extends React.Component {
                 </Tooltip>);
         }
         return (
-            <Page>
-                <PageSection variant={PageSectionVariants.light}>
-                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
-                        <Title headingLevel="h2" size="3xl">
-                            {_("Kernel crash dump")}
-                        </Title>
-                        {kdumpSwitch}
-                    </Flex>
-                </PageSection>
+            <Page
+                isBreadcrumbGrouped
+                additionalGroupedContent={
+                    <PageSection variant={PageSectionVariants.light}>
+                        <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                            <Title headingLevel="h2" size="3xl">
+                                {_("Kernel crash dump")}
+                            </Title>
+                            {kdumpSwitch}
+                        </Flex>
+                    </PageSection>
+                }>
                 <PageSection>
                     <Card>
                         <CardBody>

```


Please do the same for the kdump page.